h2o-core/src/main/java/water/network/CertAndKeyGenWrapper.java

```diff
@@ -0,0 +1,40 @@
+package water.network;
+
+import sun.security.x509.X500Name;
```

Comment:
Maybe i am wrong - it seems they can support the 'Sun' classes:
http://www-01.ibm.com/support/docview.wss?uid=isg1OA40782

Would it be possible to test that?